src/Java/Widgets/LogoWidget.java

```diff
@@ -0,0 +1,99 @@
+import vtk.vtkNamedColors;
+import vtk.vtkNativeLibrary;
+import vtk.vtkRenderWindow;
+import vtk.vtkRenderWindowInteractor;
+import vtk.vtkRenderer;
+import vtk.vtkSphereSource;
+import vtk.vtkPolyDataMapper;
+import vtk.vtkActor;
+import vtk.vtkImageCanvasSource2D;
+import vtk.vtkLogoRepresentation;
+import vtk.vtkLogoWidget;
+  
+public class LogoWidget
+{
+  // -----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+  
+    vtkNamedColors colors = new vtkNamedColors(); 
+
+    //Circle Actor Color
+    double Circlecolor[] = new double[4]; 
+    colors.GetColor("Mint", Circlecolor);
+    
```

Comment:
Use the same colours as the C++ example. This ensures consistency for users.
i.e 
``` java
double bkg[] = new double[]{0.2, 0.3, 0.4, 1.0};
int boxColor[] = new int[]{255, 127, 100, 255};
int circleColor[] = new int[4]; 
colors.GetColor("Black", circleColor);
```
So there you have your colors to use for `drawing`and `ren`. I haven't tested using `int[4]` but it should work. If not use `double[4]` and divide the ints in [255, 127, 100, 255] by 255.0.